src/calendar.cpp

```diff
@@ -63,6 +63,7 @@ moon_phase get_moon_phase( const time_point &p )
     const time_duration moon_phase_duration = calendar::season_length() / 3.0;
     //Switch moon phase at noon so it stays the same all night
     const time_duration current_day = ( p - calendar::turn_zero ) + 1_days / 2;
+    const time_point full_moon = calendar::turn_zero + calendar::season_length() / 3 / 2;
```

Comment:
Not here, in vision_test.cpp its in the tests folder.

Reply:
Ok. Sorry.